src/main/java/ru/hogwarts/school/service/StudentService.java

```diff
@@ -0,0 +1,55 @@
+package ru.hogwarts.school.service;
+
+import org.springframework.stereotype.Service;
+
+import ru.hogwarts.school.exception.FacultyNotFoundException;
+import ru.hogwarts.school.exception.StudentNotFoundException;
+import ru.hogwarts.school.model.Faculty;
+import ru.hogwarts.school.model.Student;
+import ru.hogwarts.school.repositories.StudentRepository;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+
+@Service
+public class StudentService {
+    private final StudentRepository studentRepository;
+
+    public StudentService(StudentRepository studentRepository) {
+        this.studentRepository = studentRepository;
+    }
+
+
+    public Student createStudent(Student student){
+        return studentRepository.save(student);
+    }
+    public void updateStudent(long id, Student student){
+        studentRepository.findById(id).orElseThrow(
+                () -> new StudentNotFoundException(id));
+        student.setId(id);
+        studentRepository.save(student);
+    }
+    public Student getStudent(long id){
+        return studentRepository.findById(id).orElseThrow(
+                () -> new StudentNotFoundException(id));
+    }
+    public Student removeStudent(long id){
+        Student remoteStudent = studentRepository.findById(id).orElseThrow(
+                () -> new StudentNotFoundException(id));
+
+        studentRepository.deleteById(id);
```

Comment:
```suggestion
        studentRepository.delete(remoteStudent);
```
раз достал, то лучше так